### ISSUE TITLE
Hide buttons to incomplete app functions (namely random image and filter details)

### ISSUE THREAD

**Issue by ngyikp** [deliciousblackink/Derpibooru#22]
Wouldn't it be a better idea to hide the buttons instead? I thought my Internet connection was being crap as usual, and then I looked at the source code and found out those functions are still TODOs :c

**Comment by deliciousblackink** [comment]
Yeah, I think you're right. I've been meaning to get around those for a long time now, they really shouldn't be hanging in the UI like that.
